db/schema.rb

```diff
@@ -11,11 +11,23 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
+<<<<<<< HEAD
```

Comment:
@ryanveroniwooff I believe you're supposed to remove the conflict area markers. [Check out this link on merging conflicts](https://help.github.com/articles/resolving-a-merge-conflict-from-the-command-line/). Same goes for lines 16, 18, 23, 30, and 49.